package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.1.1",
   "description": "An ES6 JavaScript Client for Salesforce Open Commerce API",
   "license": "SEE LICENSE IN LICENSE",
-  "main": "lib/index.js",
+  "main": "lib/cjs/index.js",
```

Comment:
It's not a common pattern to use `lib/cjs`.

I did some research on other open source libraries and it seems that it's a common pattern to do this:

Provide three distribution folders: `dist`, `lib` and `es`.

1. `dist` folder will contain transpiled, unified and minified output (a single UMD file).
2. `lib` folder will ship the project that is transpiled into commonJS files.
3. `es` folder will ship the source code with ES2015.

and we should use `lib/index.js` as the default output:

```json
{
  "main": "lib/index.js",
  "module": "es/index.js",
}
```